src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -99,7 +109,9 @@ export const useSwapper = () => {
   const assetIds = useSelector(selectAssetIds)
   const getSupportedSellableAssets = useCallback(
     (assets: Asset[]) => {
-      const sellableAssetIds = swapperManager.getSupportedSellableAssetIds({ assetIds })
+      const sellableAssetIds = swapperManager.getSupportedSellableAssetIds({
+        assetIds,
```

Comment:
formatting issues?

Reply:
Idk, I ran `yarn lint --fix` and this happened